packages/i18n/README.md

```diff
@@ -113,16 +113,16 @@ _Parameters_
 
 ### sprintf
```

Comment:
All of this documentation has become strictly worse. The template T is not mentioned anywhere and things appear more complex without actually being so for the most part. It's hard to see that most of these functions remain essential `string => string`.

That's more a critique of limitations to the documentation system than this work.